docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:20.04
```

Comment:
Also now that we aren't worried about syncing ubuntu version with wsl do we want this changed back to `latest`?

Reply:
might as well